packages/mosaic/sql/src/functions/list.ts

```diff
@@ -33,7 +33,7 @@ export function listContains(
   list1: ExprValue | ExprValue[],
   element: ExprValue,
 ) {
-  return fn("list_contains", exprValuesToExprNode(list1), asLiteral(element));
+  return fn("list_contains", asList(list1), asList(element));
```

Comment:
I think this should still be `asLiteral(element)`. I imagine the most common use case for this is to filter by a string, so my thought was to optimize for that, vs having it turned into a column reference, which seems very unlikely. I do understand if you want the consistency of single strings being treated as columns across the board, so whatever you think is best.

Reply:
This was a mistake. Please ignore as I fix :) I should know better than to make spot edits on GitHub rather than locally :)

Reply:
No worries, the rest of the fixes all looked great